go/interpreter/lfvm/interpreter_test.go

```diff
@@ -268,8 +267,8 @@ func TestInstructionsGasConsumption(t *testing.T) {
 	opCodes = append(opCodes, attachGasTo(10, POP_SWAP2_SWAP1_POP)...)
 	opCodes = append(opCodes, attachGasTo(4, POP_POP)...)
 	opCodes = append(opCodes, attachGasTo(6, generateOpCodesInRange(PUSH1_SHL, PUSH1_DUP1)...)...)
-	// opCodes = append(opCodes, applyGasTo(11, PUSH2_JUMP)...) // FIXME: this seems to be broken
-	opCodes = append(opCodes, attachGasTo(13, PUSH2_JUMPI)...)
+	//opCodes = append(opCodes, attachGasTo(11, PUSH2_JUMP)...)  // This test fails due to jump dest analysis
+	opCodes = append(opCodes, attachGasTo(13, PUSH2_JUMPI)...) // Jump dest analysis is skipped because argument is zero
```

Comment:
Is this hiding the "FIXME"?

I guess these tests will become all obsolete if we go for Luis' testing approach. But if not, we would miss this one. 

If this test is never going to work, please state something like "deliberately skipped because ...".

Reply:
Yes this got lost, I re-added it.